runtime/Cargo.toml

```diff
@@ -271,6 +278,62 @@ std = [
 txfilter = [
     "log"
 ]
+try-runtime = [
+    "frame-executive/try-runtime",
+    "frame-try-runtime",
+    "frame-support/try-runtime",
+
+    # For every pallet in the runtime include try-runtime
```

Comment:
You've added a `try-runtime` feature to all Zeitgeist pallets. I can see that Parity does the same thing for FRAME pallets. Why?

It doesn't seem like any pallet uses the feature. Aren't we basically adding an unused dependency here?

Reply:
At least it produces no error, when removing the `try-runtime` of all the Zeitgeist pallets. I don't really know why Parity includes this in every pallet although it's only used in a few ones. If you worry, that it would be unnecessary overhead, I see no reason to not temporarily leave that out. The two reasons to let it stay in each pallet is, that a storage migration / runtime upgrade depends on one storage item of a pallet OR the time we use it, we wouldn't have to think about including it in the runtime and `Cargo.toml` file of the pallet. 

But wouldn't cargo ignore the features anyways, when it's not activated (without `--features try-runtime`)?

[This is the PR](https://github.com/paritytech/substrate/pull/8194) where the pallets got add.

Apart from that: It think there is no dependency (overhead) added, because it comes from `frame-support` which is needed anyways.
```text
try-runtime = [
  "frame-support/try-runtime",
]
```

Reply:
Ok, I understand now. Of course we need the `try-runtime` feature if we want to use `try-runtime` in pallet-specific migrations. Ok, leave them in. :thumbsup: